src/App.tsx

```diff
@@ -113,11 +108,12 @@ function App() {
       try {
         await readFile()
         .then((content) => {
-          username = content;
+
+          username.current = content;
```

Comment:
Isn't `username` a string?

Reply:
https://stackoverflow.com/questions/57444154/why-need-useref-and-not-mutable-variable
found this is a reasoning behind why :)